### DISCUSSION TITLE
trying to find Community AMIs

### DISCUSSION THREAD

**Discussion by SRINU42** [learndevopsonline/learndevopsonline#1996] [Q&A]
Hi Raghu sir,
  i am you are 73 batch student, when i am trying launch an instance by using amis image of devops-practice it not showing AMI , if any recent update please inform me

**Comment by SRINU42** [comment]
my problem was solved

**Reply by shanmukhatadisetti** [reply]
hello Srinu, i am also facing same issue what we have to type to get AMIs image

**Reply by SRINU42** [reply]
while internet was slow , we are facing this problem, first fix the net connection

**Reply by shanmukhatadisetti** [reply]
thanks Srinu, my problem was solved

**Comment by prudhvi-1122** [comment]
Try the filter option from the image drop-down ,select the origin, hope u find this helpful

**Reply by shanmukhatadisetti** [reply]
thanks prudhvi, for your guidance
